pkg/kn/plugin/manager.go

```diff
@@ -146,11 +159,29 @@ func (manager *Manager) ListPlugins() (PluginList, error) {
 			}
 		}
 	}
+
 	// Sort according to name
 	sort.Sort(PluginList(plugins))
 	return plugins, nil
 }
 
+func isPartOfCommandGroup(commandGroupParts []string, name string) bool {
+	if commandGroupParts == nil {
+		return true
+	}
+
+	commandParts := extractPluginCommandFromFileName(name)
+	if len(commandParts) != len(commandGroupParts)+1 {
```

Comment:
Can we get some English comments for these conditions... typically not for comments but when conditions are not obvious...

Reply:
Sure.